web/src/compose_recipient.ts

```diff
@@ -170,8 +170,8 @@ function switch_message_type(message_type: MessageType): void {
     compose_ui.set_focus(opts);
 }
 
-function update_recipient_label(stream_id: number): void {
-    const stream = stream_data.get_sub_by_id(stream_id);
+function update_recipient_label(stream_id?: number): void {
+    const stream = stream_id !== undefined ? stream_data.get_sub_by_id(stream_id) : undefined;
```

Comment:
This seems like it should be a prep commit; it seems like an independent change?